CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -415,15 +418,34 @@ local function Create3DMenuItem()
 	}
 
 	local isActive = false
+	local isModal = true
```

Comment:
I am uneasy about keeping track of isModal here, it feels like we are over-coupling the menubar icons with the panels